middleware/cache/cache_test.go

```diff
@@ -231,6 +231,53 @@ func Test_Cache_NothingToCache(t *testing.T) {
 	}
 }
 
+func Test_Cache_CustomNext(t *testing.T) {
+	app := fiber.New()
+
+	app.Use(New(Config{
+		Next: func(c *fiber.Ctx) bool {
+			if c.Response().StatusCode() == fiber.StatusOK {
+				return  false
+			}
+			return true
+		},
+	}))
+
+	app.Get("/", func(c *fiber.Ctx) error {
+		return c.SendString(time.Now().String())
+	})
+
+	app.Get("/error", func(c *fiber.Ctx) error {
+		return c.Status(fiber.StatusInternalServerError).Send([]byte(time.Now().String()))
+	})
+
+	resp, err := app.Test(httptest.NewRequest("GET", "/", nil))
+	utils.AssertEqual(t, nil, err)
+	body, err := ioutil.ReadAll(resp.Body)
+	utils.AssertEqual(t, nil, err)
+
+	time.Sleep(500 * time.Millisecond)
```

Comment:
What's the meaning of Sleep here?

Reply:
At the begining, I just want to make sure to return different string. Now I found it doesn't need  to  do that. I will fix it.

Reply:
Maybe we can take advantage of CacheControl option? If it is true, we'll get extra response header if it's cached.

Reply:
I add a new commit, plz have a look.